upsertGitHubTag/deployment/index.js

```diff
@@ -120,6 +120,15 @@ function deleteEndpoint(
   req.end();
 }
 
+function handleReleaseEvent(githubEventType, body, deliveryId, path, callback) {
+  console.log("Valid release event");
```

Comment:
Doesn't seem very useful if leaving this in here, should maybe include the `deliveryId` to help someone following the logs

Reply:
Also added the delivery id to other similar `log` statements.